initfiles/bash/etc/init.d/pid.sh

```diff
@@ -195,12 +195,16 @@ check_status() {
       else
         return 1
       fi
+    elif [ $initRunning -eq 0 ]; then
+      if [ ${DEBUG} != "NO_DEBUG" ]; then
+        echo "${compName} init process not running"
+      fi
+      return 5
```

Comment:
Should update return code meanings at head of check_status too (around line 141)